src/cycax/cycad/cuboid.py

```diff
@@ -2,7 +2,20 @@
 
 
 class Cuboid(CycadPart):
+    """This class will contain information on the 3D printed part."""
+
     def __init__(self, part_no: str, x_size: float, y_size: float, z_size: float, colour="red"):
```

Comment:
FYI. The standard is to document the class not the __init__. Python is so full of these weird conventions.